test/e2e/go.mod

```diff
@@ -1,27 +1,31 @@
 module github.com/nutanix-cloud-native/cloud-provider-nutanix/test/e2e
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.1
 
 require (
-	github.com/onsi/ginkgo/v2 v2.20.2
-	github.com/onsi/gomega v1.34.2
-	k8s.io/api v0.31.1
-	k8s.io/apimachinery v0.31.1
-	sigs.k8s.io/cluster-api v1.8.3
-	sigs.k8s.io/cluster-api/test v1.8.3
-	sigs.k8s.io/controller-runtime v0.19.0
+	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/gomega v1.36.2
+	k8s.io/api v0.32.2
+	k8s.io/apimachinery v0.32.2
+	sigs.k8s.io/cluster-api v1.9.0-rc.0.0.20250228131023-67245f572cbe
+	sigs.k8s.io/cluster-api/test v1.9.0-rc.0.0.20250228131023-67245f572cbe
```

Comment:
We pull in latest changes to avoid go.mod conflicts between k8s 1.32 and clusterapi test framework 1.9.x

Reply:
Why can't we wait until there is a tagged release that references 1.32 k8s libs before we pull this in?

Reply:
because that's a month and half out and we need this for OCP.